lib/widgets/compose_box.dart

```diff
@@ -7,6 +7,8 @@ import 'package:image_picker/image_picker.dart';
 
 import '../api/model/model.dart';
 import '../api/route/messages.dart';
+import '../api/route/streams.dart';
+import '../model/autocomplete.dart' as autocomplete;
```

Comment:
This is here to avoid the ambiguity resulting from the presence of the `Autocomplete` extension in `../model/autocomplete.dart`.